web-embeds/jsconfig.json

```
@@ -1,5 +1,5 @@
 {
   "compilerOptions": {
-    "baseUrl": "."
-  }
+    "baseUrl": ".",
+  },
```

```suggestion
  }
```
I think this is why `vercel` is failing